### PR TITLE
Mobile setup repeated

### PR DESCRIPTION
**Pull request by Fatima-hub333**
Create Portfolio Website. In this task I've to create the website first two parts ("Navbar & Intro")section Via Using HTML, CSS & Flex-box model.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+.first-btn button {
+  background-color: #1a2236;
+  width: 310px;
+  height: 60px;
+  padding: 10px;
+  color: white;
+  font-size: 24px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+}
```
Comment (medaminedev66) [inline style.css]: - [x] Let's remove the border from the button by adding `border: none;` to the button selector to improve your button design.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  <div class="heading">
+  <h1>Charlie Giorgania</h1>
+  <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+    If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+  </div>
+</section>
+
+<div class="first-btn"> <button>Start collaboration</button></div>
+
+<div class="first-image"> <img src="images/2.png" alt="work-image"></div>
```
Comment (medaminedev66) [inline index.html]: - [x] The decorations are always background images, not `img` tags. Let's add the background images to the `css` file and remove them from the `html` file. Check out these links to learn how to do that: [link 1](https://www.w3schools.com/cssref/pr_background-image.asp), [link 2](https://developer.mozilla.org/en-US/docs/Web/CSS/background-image)

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @Fatima-hub333 😃, :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- ✔️The repo looks professional
- ✔️Git/GitHub flow best practices have been followed
- ✔️There are no linter errors
- ✔️You have a descriptive PR
- ✔️Your design is great and matches the Figma design

## Required Changes ♻️
- [x] Pull Request should have a descriptive title telling others what changes are inside. It would be better to change the title of your pull request to a descriptive title so your project is documented in a professional way.
- [x]  It will be nice if you could add a descriptive README file to your project. That file will give visitors a brief understanding of your project. Please use the [Microver template](https://github.com/microverseinc/readme-template) as an example.

_See the inline comments below for more required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
You can also consider:
- N/V

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by andersonlebon — APPROVED** [review]
## Hi @Fatima-hub333 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**